### PR TITLE
add the integration-tests project

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
jenkins retest

File: integration-tests/com.liferay.ide.integration.testing-repository/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/.settings
```
Comment (gamerson) [inline integration-tests/com.liferay.ide.integration.testing-repository/.gitignore]: Name this directory `com.liferay.ide.integration.tests-repository`

File: integration-tests/com.liferay.ide.integration.testing-repository/category.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<site>
+	<bundle id="com.liferay.ide.core.base" version="0.0.0" />
```
Comment (gamerson) [inline integration-tests/com.liferay.ide.integration.testing-repository/category.xml]: These bundles should have `test` in their names

`com.liferay.ide.test.core.base`
`com.liferay.ide.test.project.core.base`

File: integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/action/EnvAction.java [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+				else if (t == 4) {
+					bundleInfos[i].setVersion(value);
+				}
+			}
+		}
+
+		return bundleInfos;
+	}
+
+	private static EnvAction _envAction;
```
Comment (gamerson) [inline integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/action/EnvAction.java]: Why do we need a static instance?  can this not just be made static utilities method?    If there is `state` that needs to be kept, lets make this a junit test rule that can be shared amoung the tests.

File: integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/support/SupportBase.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+import org.junit.rules.ExternalResource;
+
+/**
+ * @author Terry Jia
+ */
+public class SupportBase extends ExternalResource {
+
+	public SupportBase() {
+		envAction = EnvAction.getInstance();
```
Comment (gamerson) [inline integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/support/SupportBase.java]: This should not use shared instance, instead it should just create a new object.

File: integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/util/ValidationMsg.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.core.base.util;
+
+/**
+ * @author Terry Jia
+ */
+public class ValidationMsg {
```
Comment (gamerson) [inline integration-tests/common/com.liferay.ide.core.base/src/com/liferay/ide/core/base/util/ValidationMsg.java]: rename to `ValidationMessage`

File: tests-resources/projects/test-liferay-workspace-ee/build.gradle [hunk 6]
```diff
@@ -0,0 +1 @@
+version = '1.0.0'
```
Comment (gamerson) [inline tests-resources/projects/test-liferay-workspace-ee/build.gradle]: Are these test resources shared between functional and integration tests?  If not, they we need to move this test resources into the integration-tests folder.

File: tests-resources/projects/test-liferay-workspace/modules/apps/sample-portlet/src/testFunctional/functions/AssertClick.function [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<definition default="assertClick">
```
Comment (gamerson) [inline tests-resources/projects/test-liferay-workspace/modules/apps/sample-portlet/src/testFunctional/functions/AssertClick.function]: We can remove all of these testFunctional files, i think those are not releveant.

**Comment by gamerson** [comment]
Hey @jtydhr88  see comments and please resend, thanks!

File: tests-resources/projects/test-liferay-workspace-ee/build.gradle (hunk 6 above)
Comment (jtydhr88) [reply tests-resources/projects/test-liferay-workspace-ee/build.gradle]: Yes, the projects or files putting in “test-resources” will be shared between functional and integration

**Pull request closed without merging**